ui/src/common/Slider.js

```diff
@@ -0,0 +1,44 @@
+import Grid from '@material-ui/core/Grid'
+import Slider from '@material-ui/core/Slider'
+import ViewModuleIcon from '@material-ui/icons/ViewModule'
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { incrementByAmount } from '../reducers/sliderSlice'
+
+const useStyles = makeStyles((theme) => ({
+  slidroot: {
```

Comment:
Please rename to `root`

Reply:
Thank you for the review; will make the required changes.